moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -0,0 +1,211 @@
+import { useMemo, useState, useCallback } from "react";
+import { useParams } from "react-router-dom";
+import { useQuery, useMutation } from "@apollo/client";
+import Box from "@mui/material/Box";
+import CircularProgress from "@mui/material/CircularProgress";
+import { DataGrid } from "@mui/x-data-grid";
+import EditOutlinedIcon from "@mui/icons-material/EditOutlined";
+import DeleteOutlineIcon from "@mui/icons-material/DeleteOutline";
+import IconButton from "@mui/material/IconButton";
+import Link from "@mui/material/Link";
+import OpenInNew from "@mui/icons-material/OpenInNew";
+import ApolloErrorHandler from "src/components/ApolloErrorHandler";
+import WorkActivityToolbar from "./ProjectWorkActivityToolbar";
+import ProjectWorkActivitiesDialog from "./ProjectWorkActivityDialog";
+import { getUserFullName } from "src/utils/userNames";
+import {
+  WORK_ACTIVITY_QUERY,
+  DELETE_WORK_ACTIVITY,
+} from "../../../../queries/funding";
+import { currencyFormatter } from "src/utils/numberFormatters";
+
+/** Hook that provides memoized column settings */
+const useColumns = ({ deleteInProgress, onDeleteActivity, setEditActivity }) =>
+  useMemo(() => {
+    return [
+      {
+        headerName: "",
+        field: "_edit",
+        hideable: false,
+        filterable: false,
+        sortable: false,
+        renderCell: ({ row }) => {
+          return deleteInProgress ? (
+            <CircularProgress color="primary" size={20} />
+          ) : (
+            <div>
+              <IconButton
+                aria-label="edit"
```

Comment:
adding `color: inherit` to this button and the one below it would make the color of the buttons match the action buttons in our other tables (the main text color)

Reply:
thanks for the tip! it woulda take me hours